crates/cargo-test-support/src/compare.rs

```diff
@@ -174,6 +176,11 @@ fn add_common_redactions(subs: &mut snapbox::Redactions) {
         regex!(r"home/\.cargo/registry/src/-(?<redacted>[a-z0-9]+)"),
     )
     .unwrap();
+    subs.insert(
+        "[VERSION]",
+        regex!(r"(?<redacted>\d+\.\d+\.\d+ \([a-f0-9]{9}+ \d{4}-\d{2}-\d{2}\))"),
+    )
```

Comment:
This feels too general.  Should we do `[CARGO_VERSION]` redacts `cargo::version()`?

Reply:
Need to convert the type.